generator/main.go

```diff
@@ -98,10 +99,15 @@ func main() {
 	command := kingpin.Parse()
 
 	parseErrors := initSNMP()
+	parseErrorCount := len(parseErrors)
 
-	if len(parseErrors) != 0 {
+	if parseErrorCount != 0 {
 		log.Warnf("NetSNMP reported %d parse errors", len(strings.Split(parseErrors, "\n")))
 	}
+	if !*failOnParseErrors {
```

Comment:
I'd expect the process to exit here

Reply:
I thought about that, but then it exits without printing the actual parse errors.